### ISSUE TITLE
Proposal: Remove Blacklight Gallery

### ISSUE THREAD

**Issue by jeremyf** [samvera/hyrax#4763]
At present there is a request to update the UI for the existing search results gallery; See #2989 

In sleuthing there are two relevant pull requests:

1. 9a0f6d1c48ad2c62e381939138158d74be0032a8 "Added grid view for search results and collections view (2014-05-08)"
2. c1246a74254d31787181f1a8e66cda67536ef646 "Add collection icon to improve masonry and slideshow views. Fixes [sufia/#2408](https://github.com/samvera-deprecated/sufia/issues/2408) (2017-01-23)"

That is the extent of the provenance and commit history.

In looking at this feature, the question is can we remove the dependency on the Blacklight Gallery gem?  We're stuck with something that fails accessibility and is not easy to upgrade (the v1.0 and v2.0 require Blacklight v7.0 which we are currently blocked in fixing).

This is a call for feedback, my preference is to remove the dependency and feature.

**Comment by jlhardes** [comment]
Looking for input @samvera/hyrax or @samvera/hyrax-repo-managers or @samvera/hyrax-ui-ux-advisors regarding the Blacklight Gallery slideshow feature for search results. See #2989 for the issue that brought this up now and screenshots showing this feature.

**Comment by mark-dce** [comment]
FWIW - we have removed the slideshow option from at least two recent projects based on stakeholder feedback and input.  In our experience, most folks do want a a list and a gallery-type view, no one that I'm aware of has had slideshow requirements.

Live repos:
Yale: https://collections.library.yale.edu/catalog?q=&search_field=all_fields&view=gallery 
UCLA: https://digital.library.ucla.edu/catalog?q=&search_field=all_fields&utf8=✓&view=gallery

**Comment by jlhardes** [comment]
Thanks for the feedback! If there is support for keeping this feature, we need to hear about it and we can keep this issue open for discussion and decisions. If no feedback comes in by the start of the next sprint (2021-03-22) to support keeping this feature, a new issue will likely be created to remove the Blacklight Gallery gem and #2989 will be closed and not fixed.

**Comment by mark-dce** [comment]
To clarify, we've worked with a number of folks who DO want the gallery view provided by the blacklight_gallery gem, just not the slide-show view.  I suspect the blacklight community has the same accessibility issues with the slideshow feature - it may be worth discussing with them whether just the slide-show functionality could be deprecated from the gem.

**Comment by jlhardes** [comment]
Thank you for that clarification! That blacklight_gallery gem is doing more than just the slideshow so I think we'll need to approach this as not removing the gem but seeing if we can modify the gem as we have it now to take out or turn off the slideshow functionality. If that ends up requiring a gem upgrade we might still have the same issues in place that @jeremyf is reporting above, but this looks like another way to approach resolving the slideshow issue in #2989.

**Comment by jlhardes** [comment]
@mark-dce can you provide information about how to disable the slideshow view in the blacklight_gallery gem or point to any documentation that already exists on how to disable that slideshow view? We are not able to update Blacklight quite yet and so we also cannot update the blacklight_gallery gem quite yet. We are contemplating pointing to documentation for disabling the slideshow view for now and then seeing what our options are for modifying the settings on that gem after we update. Thanks for any info you can provide on disabling the slideshow view!

**Comment by mark-dce** [comment]
@julesies is this what you're looking for? https://github.com/projectblacklight/blacklight-gallery/wiki/Turn-off-the-slideshow-view

**Comment by jlhardes** [comment]
Yes, thanks! I think that is something that is worth it to point out to show how to turn off the slideshow if the slideshow feature is not desired. We can review the accessibility concern in #2989 again after we are able to update Blacklight and the blacklight_gallery gem.

**Comment by jlhardes** [comment]
Closing since we will not be removing the blacklight_gallery gem. There is an accessibility concern with the slideshow controls on the current version we are using (0.7) and there is a question about the need at all for the slideshow view from that gem within Hyrax. The accessibility concern and use of the slideshow view will be revisited after updating Blacklight and the blacklight_gallery gem.
